src/anm/graphics/sheet.js

```diff
@@ -171,6 +171,22 @@ Sheet.prototype.bounds = function() {
     return new Bounds(0, 0, r[2], r[3]);
 };
 /**
+ * @method inside
+ *
+ * Checks if point is inside the image. _Does no test for bounds_, the point is
+ * assumed to be already inside of the bounds, so check `image.bounds().inside(pt)`
+ * before calling this method manually.
+ *
+ * @param {Object} pt point to check
+ * @param {Number} pt.x
+ * @param {Number} pt.y
+ * @return {Boolean} is point inside
+ */
+ Sheet.prototype.inside = function(pt) {
+    return true; // if point is inside of the bounds, point is considered to be
+                 // inside the image shape
+};
+/**
```

Comment:
All the graphics (`Path`, `Sheet`, `Text`) instances have `.inside` method from now on. For `Sheet`, in future, (when we re-fix Sprites support) it may return `false`, if this point is not located in a current region of an image. For `Text`, _may be_, it should return `false` if the given point is, i.e. between the letters, but probably it will never be done. Both these cases are enhancements, so for now they always return `true`.

And only a `Path` may return `false` for a point if it's not inside the "surface" of a curve (40% of the code in this PR :)). 

All in all, this method is for second-check if mouse event belongs to the `Element` or not, when the point is for sure already inside of a bounds. At some point it could also be improved also for `Text` and `Sheet`, but not now :).